hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -140,6 +140,17 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
             </div>
 
             <hr/>
+            <div class="form-group {{subscriptionForm.requestTimeout.$valid ? '' : 'has-error'}}">
```

Comment:
Would you mind adding a warning for the user who tries to set a request timeout over 1000ms (current default value)? It is quite high value and to comply with the fail-fast principle maybe in some cases it is better to rethink the design of a subscriber rather than increase the timeout.